packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-dashboard.spec.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import { Cloudpulse } from 'support/util/cloudpulse-page';
+import { timeUnit } from 'support/constants/time';
+import { granularity } from 'support/constants/granularity';
+import { aggregation } from 'support/constants/aggregation';
+import { timeRange } from 'support/constants/timeRange';
+import { interceptMetricsRequests } from 'support/intercepts/cloudpulseAPIHandler';
+
+
+const dashboardName = 'Linode Service I/O Statistics';
+const region = 'Chicago, IL';
+const actualRelativeTimeDuration = timeRange.Last24Hours;
+const expectedTextsCommonGranularity = [
+  granularity.Auto,
+  granularity.Min1,
+  granularity.Min5,
+  granularity.Hr1,
+  granularity.Day1,
+];
+const resource = 'test1';
+const expectedTextAgg = [aggregation.Max, aggregation.Min, aggregation.Avg];
+const expectedTextAggregationCommon = [
+  aggregation.Max,
+  aggregation.Min,
+  aggregation.Avg,
+  aggregation.Sum,
+];
+
+const cloudPulsePage = new Cloudpulse();
+
+const SDTestData = [
```

Comment:
Give meaningful name here, also since this is a constant, we can maintain this in a different file, that will hold all the constants